### ISSUE TITLE
so all your methods consider the problem as the semantic segmentation ?

### ISSUE THREAD

**Issue by jyning12** [sharon-hadar-leverate/2018-Data-Science-Bowl#1]
I observed that your label all instance as a group, so did you consider the problem as the semantic segmentation ?

**Comment by sharon-hadar-leverate** [comment]
yes, the problem could be consider as a boolean semantic segmentation, 
the DNN model outputs a value between 0 to 1, then, another threshold is used in order to convert it to 2 channels semantic segmentation - 0 is the background channel and 1 is the mask channel
